src/test/java/com/jcabi/dynamo/AwsIteratorITCase.java

```diff
@@ -47,7 +48,11 @@
     /**
      * AwsIterator can iterate.
      * @throws Exception If some problem inside
+     * @todo #56 Integration test fail when adding @NotNull constraint
```

Comment:
@tmjee the puzzle description should contains at least 20 words
Also please change `@todo #56` to `@todo #56:30min/DEV `